src/instrumentsscene/view/abstractinstrumentspaneltreeitem.cpp

```diff
@@ -258,6 +258,15 @@ void AbstractInstrumentsPanelTreeItem::setIsVisible(bool isVisible)
     }
 }
 
+void AbstractInstrumentsPanelTreeItem::setIsVisibleFromStaffTreeItem(bool isVisible)
+{
+    if (m_isVisible == isVisible) {
+        return;
+    }
+    m_isVisible = isVisible;
+    emit isVisibleChanged(isVisible);
+}
+
```

Comment:
This new method is now almost completely a copy of the normal `setIsVisible` method. It would be better to merge the methods into one, and add an extra `bool setChildren` parameter, that will determine whether children items should be made visible or invisible too. 

Because `setIsVisible` is used as a setter for a Q_PROPERTY, the new parameter will need to have a default value of `true`.

Reply:
Okay, I'll do that. Just allow me a little time :D